### ISSUE TITLE
Accuracy metric for regression documentation?

### ISSUE THREAD

**Issue by Noahyt** [tflearn/tflearn#747]
I am doing multivariable regression -- I.e. predicting a tensor with 16 float values scaled between 0 and 1.  

I have been running tests such as: 

```
[definition of network]
network = tflearn.regression(network, optimizer=optimizer_custom, loss='mean_square')
model = tflearn.DNN(network, tensorboard_dir='output/', tensorboard_verbose=3, checkpoint_path='output')
model.fit(prepared_x, prepared_t, n_epoch=1, batch_size=100,shuffle=False,snapshot_epoch=False, show_metric=True,validation_set=0.1 )
```

This has been training (I have checked the output to see it is doing what I expect. )

However, I am not sure what the "default" accuracy metric is measuring when it is displayed.  I.E. when I see in tensorboard the validation accuracy is ".75", what does this mean since my data is not categorical?  The TFLEARN documentation does not seem to address this.  

More generally, should I be using a different accuracy metric for multivariable regression?
